src/fetching/views/createNewView.ts

```diff
@@ -5,10 +5,13 @@ import { ZetkinView } from 'types/zetkin';
 import { defaultFetch } from '..';
 
 export default function createNewView(orgId: string, fetch = defaultFetch) {
-    return async () : Promise<ZetkinView> => {
+    return async (rows?: number[]) : Promise<ZetkinView> => {
```

Comment:
This might be nice to set to a default of `= []`, that way you don't have to put the or inline

Reply:
I tried making this change, but it won't work if I also want to be able to use `mutate()` without arguments (which requires the type to be `void`, which is not compatible with type of `[]`).